core/test/libraries/Volatility.t.sol

```diff
@@ -279,7 +279,7 @@ contract VolatilityTest is Test {
         uint128 tickLiquidity,
         uint32 scale
     ) public view {
-        scale = uint32(bound(scale, 1 minutes, 1 days));
+        scale = uint32(bound(scale, 1 minutes, 2 days));
```

Comment:
Why change from 1 days to 2 days?

Reply:
Referencing our offline discussion: because I changed "N" from 1 hour to 1 day, I wanted to make sure that >1 days was safe in the tests. Thus 2.